README.md

```diff
@@ -75,26 +75,24 @@ Collect and process real time twitter data to analyse popularity of tweets with
 <br>
 
 ### Domains
-
-- Data Mining
-- Data Manipulation and Feature Engineering
-- Big Data Analytics
-- Natural Language Processing
-- API and Web Server Communication
-- Network Analysis
+[![Badge](https://shields.io/badge/-Data--Mining-blue?logo=appveyor&logoColor=violet)]()&nbsp;
+[![Badge](https://shields.io/badge/-Data--Manipulatio--and--Feature--Engineering-red?logo=appveyor&logoColor=violet)]()&nbsp;
+[![Badge](https://shields.io/badge/-Big--Data--Analytics-green?logo=appveyor&logoColor=violet)]()&nbsp;
```

Comment:
color chosen here isn't appropriate.. would be better with lighter colors

Reply:
Done